data_generation/CRF2013.py

```diff
@@ -0,0 +1,1613 @@
+"""Run this via `make climate_categories/data/CRF2013.yaml` in the main
+directory."""
+
+import pathlib
+
+import climate_categories
+
+OUTPATH = pathlib.Path("./climate_categories/data/CRF2013.yaml")
+
+
+def main():
+    """Create the CRF2013 categorization from the IPCC2006 categorization, which was
+    also used by the UNFCCC to develop CRF2013. The sectors 3 (Agriculture),
+    4 (LULUCF) and 5 (Waste) are rebuilt from scratch because agriculture
+    and LUULCF are combined in one category in IPCC2006 but split in two categories
+    here. The sector codes follow the codes used in the tables where possible
+    and in consequence do not follow a consistent pattern.
+    This categorization follows the template tables (adding subcatetegories for
+    consistency in a few places). To read all submitted data an extended
+    terminology is needed for each submission year.
```

Comment:
I don't really understand the last sentence. Do you mean something like this?

```suggestion
    consistency in a few places). Because additional subcategories are often
    introduced in individual years, an extended terminology is needed for each
    submission year when reading all years.
```

Reply:
I rephrased it a bit, I hope it's clearer now. In your suggestion I didn't understand if "reading all years" refers to all submission rounds or all years of one submission round.
```suggestion
    consistency in a few places). To read all submitted data including country specific
    subsectors in "Other" categories, an extended terminology is needed. With each
    submission round (e.g. 2021) these subsectors change, thus a submission-year
    specific extended terminology might be needed to avoid cluttered sector 
    specifications.
```

Reply:
That makes it very clear for me, thanks!